src/components/TimelineComponent.tsx

```diff
@@ -14,6 +14,11 @@ const marginBottom: number = 170;
 const marginLeft: number = 70;
 const marginRight: number = 20;
 
+const scaleZoomOut = 0.9;
+const scaleZoomIn = 1.1;
+const deltaPan = 50;
+const scaleMin = 1.0;
```

Comment:
in line commenting describing the scope of these variables would be handy. For example utilized in zooming. Same goes for the rest of these long list of variables (know that was Mesa)